apps/native-component-list/screens/BarCodeScannerScreen.js

```diff
@@ -30,22 +42,47 @@ export default class BarcodeScannerExample extends React.Component {
         </View>
       );
     }
+
+    let circles = [];
+
+    if (this.state.cornerPoints != null) {
+      //console.log(this.state.cornerPoints);
+      //console.log('width, height ', this.state.width, this.state.height);
+      for (let i = 0; i < 4; ++i) {
+        let x = this.state.cornerPoints[i * 2]/this.state.width*CANVAS_WIDTH
+        let y = this.state.cornerPoints[i * 2 + 1]/this.state.height*CANVAS_HEIGHT;
+      //  console.log("zrobilem ", x, " ", y);
+        circles.push(
+          <Svg.Circle cx={x} cy={y} r={2} strokeWidth={0.1} stroke="gray" fill="green" />
+        );
+      }
+    }
     return (
-      <View style={styles.container}>
-        <BarCodeScanner
-          onBarCodeScanned={this.handleBarCodeScanned}
-          barCodeTypes={[
-            BarCodeScanner.Constants.BarCodeType.qr,
-            BarCodeScanner.Constants.BarCodeType.pdf417,
-          ]}
-          type={this.state.type}
-          style={styles.preview}
-        />
+      <ScrollView style={styles.container}>
+        <View  height={CANVAS_HEIGHT} width={CANVAS_WIDTH} >
+          <BarCodeScanner
+            height={CANVAS_HEIGHT}
```

Comment:
It would be much better if the event results don't have to be constrained to some static-width containers, let's try removing them and using flexbox 🙂